less/home-page.less

```diff
@@ -25,6 +49,14 @@
       border-radius: 20px;
       height: 100%;
     }
+    @media @phone {
+      .blast {
```

Comment:
Good job targeting the individule pictures here with their own classes. You could have also used &:not(:first-child){display: none;} here as well.

Reply:
I need to remember to dig deeper into child-selectors in CSS. I love using node selectors in JS over ID and Class selectors, so it would make sense to start doing the same thing more in CSS too.